docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,82 @@
+
+The `RedRect` is aligned to the left side of parent layout by the default value of `Target` and `RelativeOffset` properties. The `BlueRect` is aligned to the right side of parent layout by the `HorizontalAlignment`. The left side of `GreenRect` is set to be aligned with `RedRect` and right side is set to be aligned with `BlueRect`. It fills the space between `RedRect` and `BlueRect` by using `FillHorizontal`. The `YellowRect` represents that each side can be set sibling view and parent layout:
+
+**Figure 1: Preview of relative layout example with `Target` and `RelativeOffset`**
+
+![relativeLayoutExamplePreview](./media/relativeLayoutExamplePreview.png)
+
+```xaml
+<View x:Name="Parent">
+    <View.Layout>
+        <RelativeLayout/>
+    </View.Layout>
+
+    <View x:Name="RedRect"   BackgroundColor="1,0,0,1" WidthSpecification="100" HeightSpecification="100"/>
+    <View x:Name="GreenRect"  BackgroundColor="0,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="RedRect" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"
+          RelativeLayout.LeftRelativeOffset="1.0"/>
+    <View x:Name="BlueRect"  BackgroundColor="0,0,1,1" WidthSpecification="100" HeightSpecification="100"
+          RelativeLayout.RightRelativeOffset="1.0" RelativeLayout.LeftRelativeOffset="1.0"
+          RelativeLayout.HorizontalAlignment="End"/>
+    <View x:Name="YellowRect"  BackgroundColor="1,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="Parent" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"/>
+</View>
+```
+
+Since the children of relative layout are laid out depending on their relationship, you can create a responsive UI that keeps the proportions across different screen resolutions without modifications.
+width of `RectRect` and `BlueRect` are kept as `100` by their `WidthSpecification`. however, width of `GreenRect` and `YellowRect` are resized within layout space that is set by the `LeftTarget` and `RightTarget`:
+
+**Figure 2: Landscape mode of Figure 1**
+
+![relativeLayoutLandscapeExamplePreview](./media/relativeLayoutLandscapeExamplePreview.png)
+
+The following table describes the properties of `RelativeLayout`:
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LeftTarget`           | View            | Gets or sets the left side edge of the child view relative to the given target view.|
+| `RightTarget`          | View            | Gets or sets the right side edge of the child view relative to the given target view.|
+| `TopTarget`            | View            | Gets or sets the top side edge of the child view relative to the given target view.|
+| `BottomTarget`         | View            | Gets or sets the bottom side edge of the child view relative to the given target view.|
```

Comment:
| `LeftTarget`           | View            | Gets or sets the left side edge of the child view relative to the given target view.|
| `RightTarget`          | View            | Gets or sets the right side edge of the child view relative to the given target view.|
| `TopTarget`            | View            | Gets or sets the top side edge of the child view relative to the given target view.|
| `BottomTarget`         | View            | Gets or sets the bottom side edge of the child view relative to the given target view.|


Doubt: Gets or sets the bottom side edge of the child view ---------relative to the given target view.|

Is it?
 Gets or sets the bottom side edge of the child view that is relative to the given target view.|
Correct me if I am wrong.

Reply:
I have changed this to 
```
Gets or sets the target that the child's left side will refer to.
```